.github/workflows/check-maintainers-sorted.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Check that maintainer list is sorted"
+
+on:
+  pull_request_target:
```

Comment:
```suggestion
  pull_request:
```

Reply:
I'd suggest changing this back to `pull_request_target` as `pull_request` doesn't run without approval for new contributors.

[check-maintainers-sorted](https://github.com/NixOS/nixpkgs/actions/workflows/check-maintainers-sorted.yaml)

Reply:
> the full CI has to be run at least once anyway

I don't understand what this means?

> package meta checks in ofborg also need approval

I don't understand this either?

> are we missing something where approval-less runs are really useful

For new contributors this check only running if someone manually approves it (and keeps reapproving it for new pushes). It's easy to overlook that this check hasn't run and merge a PR with an entry in the wrong place which will then cause other PRs to fail. These sorts of problems are why the the other actions use pull_request_target.